java/isthmus/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
+    id("java")
+    id("idea")
+    id("com.palantir.graal") version "0.10.0"
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
```

Comment:
This not only requires a non-LTS version of Java, but also requires the use of preview features from Java 17 (IDEA throws an error by default unless you turn on preview features).

Should this be moved to a LTS version? Either 11 for broad compatibility or (just released this week) 18?

Reply:
FYI, Java 17 is a lts release.

I agree that 17 is an aggressive choice (but it's also a new project).

We are actually using some of the pattern matching that is preview in 17. We could take it out.